app/src/main/java/org/mozilla/tv/firefox/utils/ServiceLocator.kt

```diff
@@ -61,7 +62,10 @@ open class ServiceLocator(val app: Application) {
     val turboMode: TurboMode by lazy { TurboMode(app) }
     val pocketRepoCache by lazy { PocketRepoCache(pocketRepo).apply { unfreeze() } }
     val viewModelFactory by lazy { ViewModelFactory(this, app) }
-    val screenController by lazy { ScreenController() }
+    val screenControllerStateMachine by lazy {
```

Comment:
Would ScreenControllerStateMachine be used by anything other than screenController?

Reply:
It's possible that we'd want to know what the current screen is. But that might change if we're switching the ScreenControllerStateMachine to act as a pure function, rather than holding onto the current state.

Reply:
Actually I removed this because we made SCSM pure functions, no state.